integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -432,6 +433,7 @@ public static String getPodLog(String name,
                                  Integer sinceSeconds)
```

Comment:
Missing parameter description for sinceSeconds

Reply:
fixed